src/patternfly/components/InlineEdit/inline-edit.scss

```diff
@@ -0,0 +1,202 @@
+// Don't remove this magic comment. See gulpfile.js.
+// @import "../../sass-utilities/all";
+
+.pf-c-inline-edit {
+  // Value group
+  --pf-c-inline-edit__value-group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Input group
+  --pf-c-inline-edit__input-group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Toggle group
+  --pf-c-inline-edit__title-group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Action
+  --pf-c-inline-edit__action-group--item--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-inline-edit__action-group--m-inline--item--MarginRight: 0;
+
+  // Toggle
+  --pf-c-inline-edit--m-active__toggle--Color: var(--pf-global--Color--100);
+
+  // Save button
+  --pf-c-inline-edit__action--m-save--m-valid--c-button--m-plain--Color: var(--pf-global--link--Color);
+
+  // Hidden
+  .pf-m-hidden {
+    display: none;
+    visibility: hidden;
+  }
+
+  // Align right
+  .pf-m-align-right {
+    margin-left: auto;
+  }
+
+  // Overrides
+  // table {
+  //   table-layout: fixed;
+  // }
+
+  td {
+    word-break: normal;
+  }
+
+  /* stylelint-disable selector-max-compound-selectors */
+  [class*="pf-m-grid"] [data-label] {
+    .pf-c-check,
+    .pf-c-radio {
+      input {
+        grid-column: 1;
+      }
+    }
+  }
+  /* stylelint-enable */
+}
+
+// Value group
+.pf-c-inline-edit__value-group {
+  display: flex;
+
+  > * {
+    margin-right: var(--pf-c-inline-edit__value-group--item--MarginRight);
+  }
+
+  > :last-child {
+    --pf-c-inline-edit__value-group--item--MarginRight: 0;
+  }
+}
+
+// Input group
+.pf-c-inline-edit__input-group {
+  > * {
```

Comment:
why not define this as `> *:not(:last-child)` and remove the `> :last-child` selector below?

Reply:
Two reasons:
1) It leaves the `:last-child` without a declaration at all. So, if a user _did_ want to update the `:last-child`, they would have to apply both property and value `margin: var(--var);` to the selector rather than updating the variable for the `:last-child`. In a theming file, that would be inconsistent with our intent of redefining variables rather than properties and values.
2) Base specificity 10 vs 20. The property is as easily overwritable as it possibly can be. Only the `:last-child` **css variable** itself has a specificity of 20. The goal is to avoid the need for complex selectors to override basic styles/dissuade developers from using `!important` tags.

Reply:
sounds good!